Valet/Valet.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Valet.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/22/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import Foundation
+
+
+
```

Comment:
Hmm, I must have started the shim at some point and accidentally committed the new file